plugins/SitesManager/lang/en.json

```
@@ -158,27 +158,29 @@
-        "WordpressDetected": "WordPress was detected on your website. Did you know you can use the easy Matomo WordPress plugin to track your website data? %1$sLearn more in this guide%2$s.",
         "SiteWithoutDataWordpress": "WordPress",
         "SiteWithoutDataWordpressDescription": "You can use the Matomo easy WordPress plugin to track data, follow the instructions from this %1$sguide%2$s.",
-        "SiteWithoutDataWordpressFollowStep1": "Log in to your WordPress site as the administrator.",
-        "SiteWithoutDataWordpressFollowStep2": "Click on \"Plugins\" from the left menu (if you can’t see the Plugins menu, you can instead %1$sfollow these steps to insert the Matomo tracking code manually%2$s).",
-        "SiteWithoutDataWordpressFollowStep3": "Search for \"WP-Matomo\" and find the plugin named \"%1$sWP-Matomo Integration (WP-Piwik)%2$s\".",
-        "SiteWithoutDataWordpressFollowStep4": "Click \"Install Now\", wait for a moment and click on \"Activate\".",
-        "SiteWithoutDataWordpressFollowStep5": "In the left menu, click \"Settings\" -> \"WP-Matomo  \".",
-        "SiteWithoutDataWordpressFollowStep6": "In the \"Matomo Mode\" dropdown, select the option \"Self hosted (HTTP API, default)\".",
-        "SiteWithoutDataWordpressFollowStep7": "In the \"Matomo URL\" text field, enter your Matomo URL.",
-        "SiteWithoutDataWordpressFollowStep8": "%1$sClick here%2$s to create a Matomo Auth token (%3$slearn more%4$s).",
-        "SiteWithoutDataWordpressFollowStep9": "Copy the generated Auth token.",
-        "SiteWithoutDataWordpressFollowStep10": "Paste the Auth token in the “Auth token” field of the WordPress plugin.",
-        "SiteWithoutDataWordpressFollowStep11": "Confirm the \"Auto config\" checkbox is selected and click on \"Save Changes\".",
-        "SiteWithoutDataWordpressFollowStep12": "You should see \"WP-Matomo is successfully connected to Matomo\" message. If you do not see the message, make sure the values you provided in the steps above are correct, and then try again.",
-        "SiteWithoutDataWordpressFollowStep13": "Click on \"Enable Tracking\" tab.",
-        "SiteWithoutDataWordpressFollowStep14": "In the \"Add tracking code\" dropdown, select \"Default tracking\".",
-        "SiteWithoutDataWordpressFollowStep15": "Click on \"Save Changes\" and a success message will appear.",
+        "SiteWithoutDataWordpressFollowSteps1": "Log in to your WordPress site as the administrator (if you’re not an administrator %1$sclick here to send this instructions%2$s).",
+        "SiteWithoutDataWordpressFollowSteps2": "Click on \"Plugins\" from the left menu (if you can’t see the Plugins menu, you can instead %1$sinsert the Matomo tracking code manually%2$s).",
+        "SiteWithoutDataWordpressFollowSteps3": "Search for \"WP-Matomo\" and find the plugin named \"%1$s\". Note: not the \"%2$s.\" plugin.",
```

Better to remove that point, as it would get lost im some translations anyways I guess.
```suggestion
        "SiteWithoutDataWordpressFollowSteps3": "Search for \"WP-Matomo\" and find the plugin named \"%1$s\". Note: not the \"%2$s\" plugin.",
```

---

@sgiehl Updated